### ISSUE TITLE
Rasa X: result: List[Dict[Text, Any]]

### ISSUE THREAD

**Issue by kiwinesian** [RasaHQ/rasa#4115]
Hello Rasa,

First of all, thank you so much to the developers who made Rasa to be open-source! really appreciate all the hard work that went to it! :)

Just having a slight difficulty in running rasa.

I do know that perhaps Rasa X is not compatible with python3.5 as mentioned on another thread.
I upgraded and re-pointed the python to read from python3.6, but yet still having the issue as below.

Wondering if I'm missing anything that still makes Rasa still reading from python 3.5 directory?

![image](https://user-images.githubusercontent.com/12476195/61989574-9645b380-afe6-11e9-9c97-ae164eb9632c.png)

**Comment by MetcalfeTom** [comment]
Hi @kiwinesian,

Thanks for the praise 😊we love the open source community and wouldn't want to host Rasa anywhere else

To me, the above error looks like `pip` installed `rasa` into the `python3.5` directory.  Try re-installing with `python -m pip install rasa` and see if that solves the problem 👍

**Comment by kiwinesian** [comment]
thanks @MetcalfeTom for the response! :)

I think there is definitely something wrong with the `pip`
I did re-map the `pip` and the `pip3` to the python3.6 by updating the respective file at /usr/bin, but still did not work.

So what I ended up doing was install python 3.7.4 and use the pip3.7 that came with the installation to run the rasa install; works great now! :) 

Might be some incompatibility issue specific with python3.6.4? not quite sure...
The other thread that was similar was using python3.6.8
so yea.. =\
